core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2729,7 +2920,381 @@ components:
             data_struct:
               $ref: '#/components/schemas/DataStruct'
 
-
+#########################################
+# REQUEST: /lts/transaction/construction #
+#########################################
+    LtsTransactionConstructionRequest:
+      type: object
+      required:
+        - network
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+    LtsTransactionConstructionResponse:
+      type: object
+      required:
+        - epoch
```

Comment:
I wonder if we should call this `current_epoch`? What are your thoughts? 👍

Reply:
I wonder also if we should return ledger time here so people can check things are synced up.

Reply:
If by ledger time you mean the `proposer_timestamp_ms`, it might be useful indeed (what would a healthy range to check against be?). The naive way is to build the transaction with `start_epoch = current_epoch` and `end_epoch = current_epoch + 20` if you want to "shoot and forget" and don't really care about when exactly it gets committed. If you want to better control the epoch try `[current_epoch; current_epoch + 1 (or 0)]` over and over until it get's committed.